cachi2/core/package_managers/gomod.py

```diff
@@ -410,6 +413,13 @@ def run_go(_cmd: list[str], **kwargs: Any) -> str:
             )
             raise PackageManagerError(err_msg) from None
 
+    @backoff.on_exception(
+        fast_constant_delay,
+        Exception,
+        jitter=None,
+        max_tries=5,
+        logger=log,
+    )
```

Comment:
This hunk is incorrect, historically, we executed Go in 2 different modes: the one nowadays represented by `_run` dealing with go commands that don't make use of any networking and the other represented by `_retry` which we use when we fetch packages mainly. We most certainly don't want failing local go executions that have nothing to do with networking to be executed again.

Reply:
Since the commit was dropped in the meantime -> resolving.